pygfx/controllers/_orbit.py

```diff
@@ -216,20 +218,23 @@ def handle_event(self, event, viewport, camera):
                 self.rotate_stop()
             elif event.button == 2:
                 self.pan_stop()
-            viewport.renderer.request_draw()
         elif type == "pointer_move":
             xy = event.x, event.y
             if 1 in event.buttons:
                 self.rotate_move(xy),
+                if self.auto_update:
+                    viewport.renderer.request_draw()
             if 2 in event.buttons:
                 self.pan_move(xy),
-            viewport.renderer.request_draw()
```

Comment:
IIUC this is the line that caused new redraws when the mouse was moved. I didn't understand that this was caused by the controller when I first read the PR. Just marking this to make it clear :)